### DISCUSSION TITLE
opengl 10 bits mpv.conf

### DISCUSSION THREAD

**Discussion by davidav37** [dyphire/mpv-config#130] [Q&A] [answered]
hello friends,
I am seeking your knowledge because I have a problem with equipment.
I have an old AMD RX580 and for HDR in 4k films or even in 1080 p in H265 the image is dirty.
the colors are too bright, it's uglier than in h264.
so I told myself that I was going to switch to 10-bit OpenGL.
 my question is: 
how can I set mpv.conf so that it works in 10-bit opengl configuration? 
thank you very much and congratulations again for the work provided

**Comment by dyphire** [comment, marked as answer]
Just change the gpu-api option to `gpu-api=opengl`.
